pkg/plugins/policies/core/rules/resourcerules_test.go

```diff
@@ -307,6 +308,44 @@ var _ = Describe("Compute", func() {
 		Expect(rule.Conf).To(Equal([]interface{}{"conf-2"}))
 	})
 
+	It("should return Mesh rule if MeshMultiZoneService is not found", func() {
```

Comment:
Why is this related?

Reply:
Because `Compute` wasn't working. It's just the fact that a `kind: Mesh` rule applies if there's no `kind: MeshMultiZoneService`